src/test/java/com/cafe/facade/impl/order/OrderFacadeImplTest.java

```diff
@@ -0,0 +1,530 @@
+package com.cafe.facade.impl.order;
+
+import com.cafe.dto.request.OrderRegistrationRequestDto;
+import com.cafe.dto.request.OrderUpdateRequestDto;
+import com.cafe.dto.response.OrderRegistrationResponseDto;
+import com.cafe.dto.response.OrderUpdateResponseDto;
+import com.cafe.dto.response.error.ErrorOrderRegistrationResponseDto;
+import com.cafe.dto.response.error.ErrorOrderUpdateResponseDto;
+import com.cafe.entity.order.Order;
+import com.cafe.entity.order.OrderStatusType;
+import com.cafe.entity.product.ProductInOrderStatusType;
+import com.cafe.entity.table.CafeTable;
+import com.cafe.entity.table.CafeTableAssignedToWaiter;
+import com.cafe.entity.table.CafeTableStatusType;
+import com.cafe.entity.user.User;
+import com.cafe.entity.user.UserRole;
+import com.cafe.entity.user.UserRoleType;
+import com.cafe.facade.core.order.OrderFacade;
+import com.cafe.mapper.order.OrderCreationParamsMapper;
+import com.cafe.mapper.order.OrderRegistrationResponseDtoMapper;
+import com.cafe.mapper.order.OrderUpdateParamsMapepr;
+import com.cafe.mapper.order.OrderUpdateResponseDtoMapper;
+import com.cafe.service.core.order.OrderCreationParams;
+import com.cafe.service.core.order.OrderService;
+import com.cafe.service.core.order.OrderUpdateParams;
+import com.cafe.service.core.product.ProductInOrderService;
+import com.cafe.service.core.table.CafeTableAssignedToWaiterService;
+import com.cafe.service.core.table.CafeTableService;
+import com.cafe.service.core.user.UserService;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.context.TestPropertySource;
+
+import java.time.LocalDateTime;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+@ExtendWith(MockitoExtension.class)
```

Comment:
please create an AbstractTest calss, annotate it with @ExtendsWith and inherit other classes from that abstract class